web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -301,11 +294,6 @@
     }
   };
 
-  const removeAlbumsIfEmpty = async () => {
-    const albumsToRemove = ownedAlbums.filter((album) => album.assetCount === 0 && !album.albumName);
-    await Promise.allSettled(albumsToRemove.map((album) => handleDeleteAlbum(album)));
```

Comment:
We should keep this in case the user creates an empty album and then closes the tab. In the long term, it might be better to only create albums that have a title or contain assets.

Reply:
Good point, I have not considered the "tab close" case. The change has been reverted.

I had the same thought/idea about not creating the album, but if I'm not mistaken, that requires more/farther reaching changes. Maybe I can work on it in the future...